artiq/frontend/artiq_ddb_template.py

```diff
@@ -38,6 +38,16 @@ def process_header(output, description):
                 "port": 1068,
                 "command": "aqctl_corelog -p {{port}} --bind {{bind}} " + core_addr
             }},
+            "moninj": {{
+                "type": "controller",
+                "host": "{moninj_host}",
+                "port": "@rpc_port",
+                "master_addr": "{master_addr}",
+                "rpc_port": "{rpc_port}",
+                "pubsub_port": "{pubsub_port}",
+                "command": "aqctl_proxy_moninj --master-addr {{master_addr}} --bind {{bind}} "
```

Comment:
```
$ aqctl_proxy_moninj
aqctl_proxy_moninj: command not found
```
Need tweak in `setup.py` to make it callable?

Reply:
Thanks, I wasn't aware of that